can/parser_pyx.pyx

```diff
@@ -169,8 +170,8 @@ cdef class CANDefine():
       address = msg.address
       address_to_msg_name[address] = name
 
+    # Value definitions
```

Comment:
Not all signals are available in vals, plus you can't easily get signal from val.

Reply:
Can we just build v->d lookup tables to go along with the d->v tables? I looked at this awhile back because I have [some lists](https://github.com/commaai/openpilot/blob/fc8d402713f1b84b3e75067f09794236852f4f20/selfdrive/car/volkswagen/values.py#L48-L58) (with several more to come for long control) that I'd rather maintain in the DBC, but never got a round tuit.

Reply:
Ah I see, yeah that shouldn't be too hard. Probably would want to make it another PR though. I can open one

Reply:
I started looking at this real briefly back in December, I thought it was cleaner to have explicit separate maps:

```
      vd[address][sgname] = dict(zip(defs, values))
      vd[msgname][sgname] = vd[address][sgname]

      self.vd = dict(vd)
```

But I never got to the point of testing it or using it in my car port work.

Reply:
I agree it might look a little better to use, but it's so enticing being a one line change, plus I think it might make equal sense to have them in the same dict. And `dv` is already backwards...